src/addons/TextArea/TextArea.js

```diff
@@ -60,6 +60,8 @@ class TextArea extends Component {
     this.updateHeight(e.target)
   }
 
+  handleRef = c => (this.rootNode = c)
```

Comment:
Nice, we should also standardize these ref variable names.  We currently use `*Node`, `_*` and `*Ref` in various places.

A ref can either be a node (DOM component) or a reference to a class instance (Composite component).  Due to this, I vote we go with `this.*Ref` for clarity and accuracy.